client/app/bundles/course/assessment/operations/assessments.ts

```diff
@@ -52,6 +53,48 @@ export const fetchAssessmentEditData = async (
   return response.data;
 };
 
+export const fetchAssessmentLiveFeedbackSettings = (
+  assessmentId: number,
+): Operation => {
+  return async (dispatch) => {
+    return CourseAPI.assessment.assessments
+      .liveFeedbackSettings(assessmentId)
+      .then((response) => {
+        dispatch(
+          saveAllAssessmentsQuestions({
+            assessments: response.data.assessments,
+            tabs: [],
+            categories: [],
```

Comment:
any reason why tabs / categories are set to empty array?

Reply:
actually we couldn't care less about the tabs and categories; this is for assessmentEdit only.. However, the input of tabs and categories are required to be stored in redux, and hence it's technically can be set to anything. I just put empty array there.

Reply:
I ended up refactoring the action call for this (so now tabs and categories can be undefined, and in that case, we don't update tabs and categories on redux)